tests/unit/spec/base/core-datalayer.js

```diff
@@ -0,0 +1,27 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect, sinon, spyOn, jasmine */
+
+describe('core-datalayer.js', function() {
+
+    describe('PageId', function(){
+
+        it('will grab data-gtm-page-id value if present on <html> element', function(){
+            $('html').attr('data-gtm-page-id', 'test');
```

Comment:
Curious, does it really matter if I setup the `removeAttribute` call in an `afterEach` call or could I just do it in the spec since I only add it in the first spec?